### ISSUE TITLE
Support for clap version 4

### ISSUE THREAD

**Issue by plugwash** [maidsafe/resource_proof#47]
Hi.

I am one of the Debian rust team and we would like to reduce the number of clap versions we are shipping. Is there any chance of an update to support clap version 4?

**Comment by dirvine** [comment]
Absolutely, if you fancy a PR we can merge that or we can put it in our queue for update.
